core/aws_ddk_core/stages/athena_sql.py

```diff
@@ -65,8 +68,11 @@ def __init__(
             Identifier of the stage
         environment_id : str
             Identifier of the environment
-        query_string : str
+        query_string : Optional[str]
             SQL query that will be started
+        query_string_path : Optional[str]
+            dynamic path in statemachine for SQL query to be started
+            `$.queryString` by default
```

Comment:
I don't see this as a default any where in the codebase

Reply:
It was previously in my design but i decided against it, fixed this in code